isis/src/base/apps/fits2isis/tsts/default/Makefile

```diff
@@ -4,10 +4,6 @@ include $(ISISROOT)/make/isismake.tsts
 
 commands:
 	$(APPNAME) from=$(INPUT)/WFPC2u5780205r_c0fx.fits \
-	to=$(OUTPUT)/fitsTruth.cub \
-	> /dev/null;
-	raw2isis from=$(INPUT)/WFPC2u5780205r_c0fx.fits \
```

Comment:
Why did you remove this raw2isis call?

Reply:
We don't do anything after the raw2isis call, so there is no need for it. The only reason I could see why we would call raw2isis is to check if the two output cubes are the same.